### ISSUE TITLE
The download link of GTFS is invalid in reader-gtfs/README.md

### ISSUE THREAD

**Issue by guqing** [graphhopper/graphhopper#2639]
The GTFS download link is invalid in https://github.com/graphhopper/graphhopper/blob/master/reader-gtfs/README.md
![image](https://user-images.githubusercontent.com/38999863/182160009-604eae41-c8e2-46a6-b1ab-0d9a243ce2a3.png)

I got a 404 error
```
➤ wget -O gtfs-vbb.zip https://www.vbb.de/fileadmin/user_upload/VBB/Dokumente/API-Datensaetze/GTFS.zip

--2022-08-01 21:39:03--  https://www.vbb.de/fileadmin/user_upload/VBB/Dokumente/API-Datensaetze/GTFS.zip
Resolving www.vbb.de (www.vbb.de)... 198.18.1.56
Connecting to www.vbb.de (www.vbb.de)|198.18.1.56|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-08-01 21:39:04 ERROR 404: Not Found.
```

/kind document

**Comment by easbar** [comment]
Yes, VBB announced they changed the URL **and** the dataset (in German): https://www.vbb.de/vbb-services/api-open-data/datensaetze/

The new URL is: https://www.vbb.de/fileadmin/user_upload/VBB/Dokumente/API-Datensaetze/gtfs-mastscharf/GTFS.zip

The announcement says that the `stop_id` in `stop_times.txt`, `stops.txt` and `transfers.txt` now uses the 'DHID' (Germany stop ID) and that each `stop_id` carries its own coordinates and no longer references the coordinates of the `parent_station`. 

Can you try if this works for you? I'm not sure if we need to adopt to the changed data format, but probably not.

**Comment by guqing** [comment]
> Yes, VBB announced they changed the URL **and** the dataset (in German): https://www.vbb.de/vbb-services/api-open-data/datensaetze/
> 
> The new URL is: https://www.vbb.de/fileadmin/user_upload/VBB/Dokumente/API-Datensaetze/gtfs-mastscharf/GTFS.zip
> 
> The announcement says that the `stop_id` in `stop_times.txt`, `stops.txt` and `transfers.txt` now uses the 'DHID' (Germany stop ID) and that each `stop_id` carries its own coordinates and no longer references the coordinates of the `parent_station`.
> 
> Can you try if this works for you? I'm not sure if we need to adopt to the changed data format, but probably not.

OK, thanks for your reply, I will try it
